content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -11,7 +11,7 @@ weight: 65
 
 {{< feature-state for_k8s_version="v1.27" state="alpha" >}}
 
-Dynamic resource allocation is a new API for requesting and sharing resources
+Dynamic resource allocation is a API for requesting and sharing resources
```

Comment:
Should this be "an API" instead of "a API"?

Reply:
```suggestion
Dynamic resource allocation is an API for requesting and sharing resources
```